config/configmap/inferenceservice.yaml

```diff
@@ -24,12 +24,22 @@ data:
             "multiModelServer": "false"
         },
         "sklearn": {
+          "v1": {
             "image": "gcr.io/kfserving/sklearnserver",
             "defaultImageVersion": "v0.4.1",
             "supportedFrameworks": [
               "sklearn"
             ],
             "multiModelServer": "false"
+          },
+          "v2": {
+            "image": "docker.io/seldonio/mlserver",
+            "defaultImageVersion": "0.1.2",
+            "supportedFrameworks": [
+              "sklearn"
+            ],
+            "multiModelServer": "false"
```

Comment:
@adriangonz Can you help add these config to the test overlay as well https://github.com/kubeflow/kfserving/blob/master/config/overlays/test/configmap/inferenceservice.yaml ? The CI crashed on 

```
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/kubeflow/kfserving/pkg/apis/serving/v1beta1.(*SKLearnSpec).getContainerV1(0xc0008a7080, 0xc0008be2e0, 0x16, 0x0, 0x0, 0xc0008be300, 0x15, 0xc000453b20, 0x6c, 0xc000280c30, ...)
	/go/src/github.com/kubeflow/kfserving/pkg/apis/serving/v1beta1/predictor_sklearn.go:86 +0x356
github.com/kubeflow/kfserving/pkg/apis/serving/v1beta1.(*SKLearnSpec).GetContainer(0xc0008a7080, 0xc0008be2e0, 0x16, 0x0, 0x0, 0xc0008be300, 0x15, 0xc000453b20, 0x6c, 0xc000280c30, ...)
	/go/src/github.com/kubeflow/kfserving/pkg/apis/serving/v1beta1/predictor_sklearn.go:68 +0xb3
github.com/kubeflow/kfserving/pkg/controller/v1beta1/inferenceservice/components.(*Predictor).Reconcile(0xc000926180, 0xc0009d7000, 0x1, 0x1)
```

Reply:
Done!